code/game/gamemodes/antag_spawner.dm

```diff
@@ -205,7 +205,36 @@
 
 
 ///////////SLAUGHTER DEMON
+/obj/item/weapon/antag_spawner/slaughter_demon
+	name = "Bottle of Blood"
+	desc = "A bottle of magically infused blood, the smell of which will attract extradimensional beings when broken."
+	icon = 'icons/obj/wizard.dmi'
+	icon_state = "vial"
+	
+/obj/item/weapon/antag_spawner/slaughter_demon/proc/check_usability(mob/user)
+	if(used)
+		user << "<span class='warning'>Somehow the bottle has already been emptied!</span>" // shouldn't happen as it destroys itself upon succesful summoning
+		return 0
+	if(user.z == ZLEVEL_CENTCOM)
+		user << "<span class='warning'>There are no demons within this realm. Try using the [src] onboard the station.<span>"
+		return
+	return 1
 
+/obj/item/weapon/antag_spawner/slaughter_demon/attack_self(mob/user)
+	if(!(check_usability(user)))
+		return
+
+	var/list/wizard_candidates = get_candidates(ROLE_WIZARD, 3000, "slaughter demon") // best pref for slaughter demon, as it isn't a pref on it's own.
+	if(wizard_candidates.len > 0)
+		used = 1
+		var/client/C = pick(wizard_candidates)
+		spawn_antag(C, get_turf(loc), "Slaughter Demon")
+		new /obj/effect/gibspawner/human(loc)
+		playsound(loc, 'sound/effects/Glassbr2.ogg', 25, 1)
+		user << "<font size=3><span class='danger'><b>\The bottle sizzles and the bottle shatters as the contents splatter over the floor, only to burn away moments later. You feel a sense of dread wash over you as the contents dissapear without trace.\</font></span>"
```

Comment:
also there's a \ at the end which isn't needed apparently. and maybe you'd like this message to be displayed to everyone on sight so visible_message() instead of user <<